aerogear/aerogear-android-push
# Agdroid 475 refresh token

How to test : 
 * mvn clean install with device attached
 * make sure you have core and pipe 3.0.0-SNAPSHOT installed
 * You should see no errors

Bonus Tests : 
 Check out the jboss helloworlds and apply the changes from y previous PR (I can lend a hand upon request)

Blockers : 
 I don't know how to get Google to send the token changed Intent.  Any thoughts on how to test that part?

 Possible improvements:

I feel like there is some refactoring to be had.
